### ISSUE TITLE
what's the different between LightFM and tensorrec?

### ISSUE THREAD

**Issue by letotefrank** [jfkirk/tensorrec#92]
hi,jfkirk！Can you answer this question in detail? 
I have used LightFM to build a recommendation system. Is it necessary to use this(tensorrec)?

**Comment by jfkirk** [comment]
Hey @letotefrank 

LightFM and TensorRec are similar packages. I used LightFM quite a bit before writing TensorRec.

LightFM is a great system for learning linear embeddings from user/item metadata. The project is mature, and the package is quite effective at problems where a linear embedding is sufficient.

TensorRec gives you the option to customize your recommender systems embedding (aka representation) functions. This allows you to apply deep nets, non-linear transforms, or anything else you can implement in TensorFlow as embedding functions. By default, TensorRec is very similar to LightFM.

If you don't have a particular need to use non-linear representation functions,  custom prediction functions, or custom loss functions, then you're probably best served using LightFM for now.

Hope this helps!
